tf_module/ckr_gcp/main.tf

```diff
@@ -68,6 +68,7 @@ resource "google_cloudfunctions_function" "key_rotator_cloud_function" {
   trigger_http          = true
   entry_point           = "Request"
   service_account_email = google_service_account.key_rotator_service_account.email
+  timeout = var.ckr_timeout
```

Comment:
The linter's going to complain about this one - would you mind running `terraform fmt` over the module?

Reply:
The formatting should be correct now